client/my-sites/plans-features-main/index.jsx

```diff
@@ -152,6 +152,14 @@ export class PlansFeaturesMain extends Component {
 				) }
 				data-e2e-plans={ displayJetpackPlans ? 'jetpack' : 'wpcom' }
 			>
+				{ /* @todo: Add translations in FormattedHeader once the final copy is provided. */ }
+				{ isEnabled( 'plans/jetpack-backup' ) && (
```

Comment:
We should also add a `&& isJetpack` here, because currently this will also appear for WP.com sites:

![](https://cldup.com/pVvuSPQFQM.png)

Reply:
Should we then use the even more specific: `displayJetpackPlans`?

Reply:
Sounds good to me, let's use that!